vunit/rivierapro_interface.py

```diff
@@ -135,13 +135,33 @@ def compile_source_file_command(self, source_file):
         LOGGER.error("Unknown file type: %s", source_file.file_type)
         raise CompileError
 
+    @staticmethod
+    def _std_str(vhdl_standard):
+        """
+        Convert standard to format of Riviera-PRO command line flag
+        """
+        if vhdl_standard == "2019":
```

Comment:
Same suggestion as [above](https://github.com/VUnit/vunit/pull/547/files#r327522796). However, an exception is required here, since `2019` is converted to `-2018`.